code/game/objects/items/storage/backpack.dm

```diff
@@ -468,6 +468,7 @@
 	new /obj/item/clothing/shoes/sneakers/black(src)
 	new /obj/item/storage/belt/security/deputy(src)
 	new /obj/item/clothing/accessory/armband/deputy(src)
+	new /obj/item/card/id/pass/minisec(src)
```

Comment:
The Deputy bag doesn't come with Deputy accesses because the Warden/HoS has a box of them.

Keep in mind that Security Officers can buy things from the vending machine, and Sec Officers shouldn't be able to promote deputies.